cmd/acra-server/acra-server.go

```diff
@@ -313,7 +320,21 @@ func main() {
 
 		ocspDbVerifier := network.DefaultOCSPVerifier{Config: *ocspDbConfig, Client: &network.DefaultOCSPClient{}}
 
-		dbTLSConfig, err = network.NewTLSConfig(network.SNIOrHostname(*tlsDbSNI, *dbHost), *tlsDbCA, *tlsDbKey, *tlsDbCert, tls.ClientAuthType(*tlsDbAuthType), ocspDbVerifier, crlVerifier)
+		var crlDbConfig *network.CRLConfig
+		if len(*tlsCrlDbUrl) > 0 {
```

Comment:
same here. it's okay to check by `len` if we will split values to an array of strings by some separator.